src/k8s/subscriptions/UAT-CSTAR/terraform.tfvars

```diff
@@ -441,6 +441,210 @@ autoscaling_specs = {
     ]
   }
 }
+
+fa_austoscaling_specs = {
```

Comment:
@antoniocapizzi95 Please check variable name. As far as I can understand, fa_au**s**toscaling_specs is wrong